runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -5845,6 +5845,13 @@ class INTERPRETER_CLASS
 			}
 			break;
 		}
+		case J9VM_CONTINUATION_RETURN_FROM_SYNC_METHOD_JNI: {
+			UDATA *bp = (UDATA*)_sp + 4;
+			_sendMethod = *(J9Method**)_sp;
```

Comment:
surrounding code uses different formatting
```suggestion
			UDATA *bp = (UDATA *)_sp + 4;
			_sendMethod = *(J9Method **)_sp;
```